prime-router/build.gradle.kts

```diff
@@ -288,6 +290,16 @@ tasks.register<Test>("testIntegration") {
     }
 }
 
+tasks.register<ResolveTask>("generateOpenApi") {
```

Comment:
We should run this automatically as part of the build process, Instead of having to do it manually. Should we be committing these files to the repo? My vote would be no, so maybe we should put these in the build directory? If we are going to host docs eventually, where is the best place to put them?

Reply:
Agree, the generated spec (yaml file) is targeted in a repo folder - ./prime-router/docs/api/generated/api.yaml

I think that's temporary and it should be target to some where under build folder, will make the change.

As for deploy the API docs, I haven't thought about it yet.

It should be some sites where the RS API consumers (external, internal developers?), for CMS bluebutton API, there is a sandbox deployment (public) and the swagger UI docs can be access there (through an endpoint), and the calls are wired to the BB2 server on sandbox - using synthetic data.